schemas/questionnaire_v1.json

```diff
@@ -55,8 +55,12 @@
       "description": "The survey (Census or Census Coverage Survey) that an address has been sampled into for participation"
     },
     "form_type": {
-      "enum": ["H", "I", "C"],
-      "description": "The form types are Household (H), Individual (I) or Communal Establishment (C)"
+      "enum": ["H", "I", "C", "business"],
+      "description": "The form types are Household (H), Individual (I), Communal Establishment (C) and business"
+    },
+    "eq_id": {
+      "enum": ["test"],
+      "description": "The eq id, currently only test"
```

Comment:
An `eq_id` can be any alphanumeric string. The current business surveys can be seen at https://github.com/ONSdigital/eq-survey-runner/tree/master/data/en

Reply:
Done, thanks!